pythainlp/benchmarks/word_tokenisation.py

```diff
@@ -1,310 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
- [ ] Add benchmarks

Reply:
That was my mistake, I already fixed it. 😥